### PR TITLE
[MNOE-549] Add tasks controllers, models and routes

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

**Comment by hedudelgado** [comment]
@x4d3 @ouranos

File: api/app/controllers/mno_enterprise/jpi/v1/admin/orga_relations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        @orga_relations = @orga_relations.where(params[:where]) if params[:where]
+        @orga_relations = @orga_relations.all.fetch
+        response.headers['X-Total-Count'] = @orga_relations.metadata[:pagination][:count]
+      end
+    end
+
+    # GET /mnoe/jpi/v1/admin/orga_relations/1
+    def show
+      orga_relation
+      render_not_found('orga relation') unless @orga_relation
```
Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/orga_relations_controller.rb]: why not directly ?
```def show
  render_not_found('orga relation') unless orga_relation
end```

File: api/app/controllers/mno_enterprise/jpi/v1/admin/tasks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        render 'show'
+      else
+        render_bad_request('update task', task.errors)
+      end
+    end
+
+    private
+
+    def tasks
+      if params[:outbox] == 'true'
```
Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/tasks_controller.rb]: why no directly`if params[:outbox]` ?

File: api/app/controllers/mno_enterprise/jpi/v1/admin/tasks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        @tasks ||= MnoEnterprise::Task.where('task_recipients.orga_relation_id'=> current_user.organizations.first.orga_relation_id)
+      end
+    end
+
+    def task
+      @task ||= MnoEnterprise::Task.find(params[:id].to_i)
+    end
+    
+    def send_task
+      params[:task][:status] == 'sent'
```
Comment (x4d3) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/tasks_controller.rb]: `task_sent` ?

**Comment by x4d3** [comment]
@ouranos  I am going to merge this branch in order to be able to test. You can still comment on it, and we will review when we merge on 3.3.

**Pull request merged**